CMakeLists.txt

```diff
@@ -1159,6 +1159,21 @@ if(CONFIG_BUILD_OUTPUT_BIN)
   endif()
 endif()
 
+if(CONFIG_BUILD_OUTPUT_BIN AND CONFIG_BUILD_OUTPUT_UF2)
```

Comment:
Why do we need both conditions in the if.  How can UF2 be set w/o BIN?

Reply:
It can't really, since the Kconfig depends on it. I can change the conditional.

Reply:
The reason I'd added was to be explicit, since we invoke and pass in the bin output to the script.